extensions/sql-migration/src/dialog/skuRecommendationResults/SelectStorageAccountDialog.ts

```diff
@@ -0,0 +1,608 @@
+			this._targetSubscription = (selectedSubscription)
+				? utils.deepClone(selectedSubscription)!
+				: undefined!;
+
+		} catch (e) {
+			console.log(e);
+		} finally {
+			this._azureSubscriptionDropdown.loading = false;
+		}
+	}
+
+	private async _populateLocationDropdown(): Promise<void> {
+		try {
+			this._azureLocationDropdown.loading = true;
+			this._storageAccounts = await utils.getStorageAccounts(
+				this._azureAccount,
+				this._targetSubscription
+			);
+			this._locations = await utils.getResourceLocations(
+				this._azureAccount,
+				this._targetSubscription,
+				this._storageAccounts);
+
+			this._azureLocationDropdown.values = utils.getAzureLocationsDropdownValues(this._locations);
+			utils.selectDefaultDropdownValue(
+				this._azureLocationDropdown);
+			const selectedLocation = this._locations?.find(location => location.displayName === (<azdata.CategoryValue>this._azureLocationDropdown.value)?.displayName);
+			this._location = (selectedLocation)
+				? utils.deepClone(selectedLocation)!
+				: undefined!;
+		} catch (e) {
+			console.log(e);
+		} finally {
+			this._azureLocationDropdown.loading = false;
+		}
+	}
+
+	private async _populateResourceGroupDropdown(): Promise<void> {
+		try {
+			this._azureResourceGroupDropdown.loading = true;
+
+			this._resourceGroups = utils.getServiceResourceGroupsByLocation(
+				this._storageAccounts,
+				this._location);
+
+			this._azureResourceGroupDropdown.values = utils.getResourceDropdownValues(
+				this._resourceGroups,
+				constants.RESOURCE_GROUP_NOT_FOUND);
+
+			utils.selectDefaultDropdownValue(
+				this._azureResourceGroupDropdown);
+
+			const selectedResourceGroup = this._resourceGroups?.find(rg => rg.name === (<azdata.CategoryValue>this._azureResourceGroupDropdown.value)?.displayName);
+			this._resourceGroup = (selectedResourceGroup)
+				? utils.deepClone(selectedResourceGroup)!
+				: undefined!;
+		} catch (e) {
+			console.log(e);
+		} finally {
+			this._azureResourceGroupDropdown.loading = false;
+		}
+	}
+
+	private async _populateBlobStorageAccount(): Promise<void> {
+		try {
+			this._azureStorageAccountDropdown.loading = true;
+
+			this._storageAccounts = await utils.getStorageAccounts(
+				this._azureAccount,
+				this._targetSubscription
+			);
+
+
+			this._azureStorageAccountDropdown.values = utils.getAzureResourceDropdownValues(
+				this._storageAccounts,
+				this._location!,
+				this._resourceGroup?.name,
+				constants.NO_STORAGE_ACCOUNT_FOUND);
+
+			utils.selectDefaultDropdownValue(
+				this._azureStorageAccountDropdown
+			);
+
+			const selectedBlobStorage = this._storageAccounts?.find(rg => rg.name === (<azdata.CategoryValue>this._azureStorageAccountDropdown.value)?.displayName);
+			this._storageAccount = (selectedBlobStorage)
+				? utils.deepClone(selectedBlobStorage)!
+				: undefined!;
+
+		} catch (error) {
+			logError(TelemetryViews.UploadArmTemplateDialog, '_populateStorageAccount', error);
```

Comment:
some where we are using logError and somewhere console.log to capture errors. Let's make it uniform. I will say use logError everywhere.